src/library/scala/collection/Iterable.scala

```diff
@@ -705,7 +705,7 @@ trait IterableOps[+A, +CC[_], +C] extends Any with IterableOnce[A] with Iterable
   })
 
   /** Alias for `concat` */
-  @`inline` final def ++ [B >: A](suffix: IterableOnce[B]): CC[B] = concat(suffix)
+  def ++ [B >: A](suffix: IterableOnce[B]): CC[B] = concat(suffix)
```

Comment:
I think we should at least have a `@deprecatedOverriding` annotation explaining that this method must be an alias to `concat`, and that `concat` is usually the method you want to override.